imagecraft/models/project.py

```diff
@@ -226,16 +187,55 @@ def _validate_all_platforms(cls, platforms: dict[str, Any]) -> dict[str, Platfor
 
         return platforms
 
+class Project(YamlModelMixin, BuildPlanner, BaseProject):
+    """Definition of imagecraft.yaml configuration."""
+
+    series: str
```

Comment:
I think this is good for now, but per our recent discussions we really need to see about dropping `series` and just building per `build_base`. Yes, this limits us to only building series that are equal to the build environment, but this is already a constraint in livecd-rootfs - and in theory in other crafts too.